runtime/service/src/test/java/org/apache/polaris/service/events/listeners/aws/cloudwatch/AwsCloudWatchEventListenerTest.java

```diff
@@ -236,12 +248,18 @@ void shouldSendEventToCloudWatch() {
           .satisfies(
               logEvent -> {
                 String message = logEvent.message();
+                JsonNode root = objectMapper.readTree(message);
+                JsonNode event = root.path("event").isMissingNode() ? root : root.path("event");
```

Comment:
nit: shouldn't the @JsonUnwrapped make sure that the event information is not under an "event" object?